Makefile

```diff
@@ -96,3 +96,149 @@ docker-httpserver: ## Build the HTTP server Docker image
 		--file httpserver.dockerfile \
 		--tag your-project \
 	.
+
+##@ Packaging
+
+.PHONY: package-build
+package-build: ## Build packages (without releasing)
+	@echo "Building packages..."
+	@goreleaser build --snapshot --clean
+	@echo "✅ Packages built in dist/"
+
+.PHONY: package-local
+package-local: ## Build packages locally for testing
+	@echo "Creating local release packages..."
+	@goreleaser release --snapshot --clean
+	@echo "✅ Release packages created in dist/"
+	@echo "📦 Created packages:"
+	@find dist/ -name "*.deb" -o -name "*.rpm" -o -name "*.tar.gz" | sort
+
+.PHONY: package-test-reproducible
+package-test-reproducible: ## Test reproducible builds
```

Comment:
this should really be a shell script, because so much is happening. as a rule of thumb, if you use `if` or functions then make it a bash script instead and just call from here.